organization/components/roles/RoleTable.tsx

```
@@ -0,0 +1,308 @@
+                                <button
+                                  onClick={e => {
+                                    e.stopPropagation();
+                                    setDeleteConfirm(role._id);
+                                  }}
+                                  className="inline-flex items-center px-2.5 py-1.5 border border-gray-300 shadow-sm text-xs font-medium rounded text-gray-700 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 transition-colors"
+                                >
+                                  <Trash2 className="h-3 w-3 mr-1 text-gray-400" />
+                                  Delete
+                                </button>
+                              )}
+                            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)